src/hdskins/java/com/voxelmodpack/hdskins/HDSkinManager.java

```diff
@@ -87,12 +87,10 @@
     private List<ISkinModifier> skinModifiers = Lists.newArrayList();
 
     private SkinResourceManager resources = new SkinResourceManager();
-    //    private ExecutorService executor = Executors.newCachedThreadPool();
```

Comment:
If you don't plan on uncommenting this, remove it.

Reply:
But I did remove it? You're the one that was leaving commented code all over the place.

Reply:
Oops. Looks like I was color blind for a bit.

Reply:
I'll forgive than then ^^